### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by wtrocki**
## What was done

Added logging and debug.

### CODE REVIEW

File: web/src/sdk/links/debugLink.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        // data from a previous link
+        const time = new Date() - operation.getContext().start;
+        logger(`operation ${operation.operationName} took ${time} to complete`);
+        return data;
+    })
+});
+
+export const requestTimeLink = concat(timeStartLink, logTimeLink)
+
+export class DebugLink extends ApolloLink {
```
Comment (danielpassos) [inline web/src/sdk/links/debugLink.js]: Why not wrapper the real link inside of this and use the `requestTimeLink` only over here?

**Comment by danielpassos** [comment]
I don't see the _conflict_ on the console anymore also the _retry_ stopped to work.

**Comment by wtrocki** [comment]
Still in progress.. Feel free to ignore this PR. I was experimenting a lot..

**Comment by wtrocki** [comment]
I will look forward for opinions on this. I kinda do not like the Generic Link I have written to visualize flow - we can add more logging inside each link to support that rather than adding some dumb links between ?
WDYT?

File: web/src/sdk/links/debugLink.js (hunk 1 above)
Comment (wtrocki) [reply web/src/sdk/links/debugLink.js]: Yeah.. This is something I should address good point.

**Comment by StephenCoady** [comment]
Huge +1 from me on having some kind of thread we can pull through our links. I think you are right though and possibly a file we can import into each link manually might be better as this won't pollute the link chain we pass to the client.

**Comment by danielpassos** [comment]
Well, I like the idea but not sure about the implementation. I really don't have a strong opinion about having a `debugLink` and it wrapper the real `*link` to track whatever we want or create a `baseLink` and all link extend of this.

**Comment by wtrocki** [comment]
Yeah... Having extensive logging inside our links can deliver more value and insight rather than generic links between. Going to remove this and this will also remove other problems.

**Comment by wtrocki** [comment]
Removed unneeded links. 
@danielpassos This stopped logging by default because we need to explicitly enable logging.

**Comment by wtrocki** [comment]
To enable logging: `localStorage.debug = 'AeroGearSync:*'`

File: web/src/sdk/mutations/QueueMutationLink.js [hunk 2]
```diff
@@ -92,4 +72,36 @@ export class QueueMutationLink extends ApolloLink {
     this.queue = []
   }
 
+  /**
+   * Merge offline operations that are made on the same object.
+   * Equality of operation is done by checking operationName and object id.
+   */
+  squashOperations(query, operationName, variables) {
+    let operationName
```
Comment (StephenCoady) [inline web/src/sdk/mutations/QueueMutationLink.js:80]: duplicate declaration here

**Review by StephenCoady — APPROVED** [review]

**Pull request merged**